projects/packages/jetpack-mu-wpcom/src/features/help-center/class-help-center.php

```diff
@@ -100,7 +100,8 @@ function ( $wp_admin_bar ) {
 							),
 						)
 					);
-				}
+				},
+				12
```

Comment:
Is this priority change to ensure its left/right position wrt to the new help item?

Reply:
@Addison-Stavlo Yes! I guess [my earlier comment](https://github.com/Automattic/jetpack/pull/38976#discussion_r1723884966) disappeared. Previously it was the default priority, 10. To make sure they're next to each other, I gave Reader an 11 priority and Help Center a 12 priority. Let me know if you have another recommendation!

Reply:
Not off the top of my head, that makes sense to me!

Perhaps a comment to help context for the next code reader would be helpful.

Reply:
Added, thanks!